prime-router/src/main/kotlin/Sender.kt

```diff
@@ -41,6 +44,34 @@ open class Sender(
         }
     }
 
+    /**
+     * @property testManufacturer Sender a test manufacturer (Abbott, Roche, Quidel)
+     * @property dataAggregator Sender is a data aggregator (LifePoint, ImageMover, SimpleReport, Prescryptyve)
+     * @property facility Sender is a facility (CSV-uploader, urgent care, nursing homes, A1 Health--home health and hospice)
+     * @property hospitalSystem Sender is a hospital or large hospital system (HCA)
+     */
+    enum class SenderType {
+        @JsonProperty("testManufacturer")
```

Comment:
Remove all these @JsonProperty tags.  They are only used if the name of the variable is different.

Reply:
Please be specific about this detail: "if the name of the variable is different" - different than what?